pages/common/pdfjoin.md

```diff
@@ -1,15 +1,20 @@
 # pdfjoin
 
-> PDF merging utility.
+> PDF merging utility based on pdfjam.
+> More information: <https://github.com/rrthomas/pdfjam-extras>.
 
-- Merge two PDFs:
+- Merge two PDFs into one with the default suffix "joined":
 
-`pdfjoin {{file1}} {{file2}} --outfile {{output_file}}`
+`pdfjoin {{path/to/file1.pdf}} {{path/to/file2.pdf}}`
 
-- Save pages 3 to 5 followed by page 1 to a new PDF:
+- Merge the first page of each given file together:
 
-`pdfjoin {{file 3-5,1}} --outfile {{output_file}}`
+`pdfjoin {{path/to/file1.pdf path/to/file2.pdf ... 1}} --outfile {{output_file}}`
```

Comment:
```suggestion
`pdfjoin {{path/to/file1.pdf path/to/file2.pdf ...}} {{1}} --outfile {{output_file}}`
```

Reply:
I'm not sure I understand the way you've used the syntax in this new example. Could you elaborate please, @xeruf?